jwst/flatfield/flat_field.py

```diff
@@ -1119,9 +1174,28 @@ def combine_dq(f_flat_dq, s_flat_dq, d_flat_dq, default_shape):
         flat_dq = dq_list[0].copy()
     elif n_dq == 2:
         flat_dq = np.bitwise_or(dq_list[0], dq_list[1])
+        bad1 = np.bitwise_and(dq_list[0], dqflags.pixel['DO_NOT_USE'])
+        bad2 = np.bitwise_and(dq_list[1], dqflags.pixel['DO_NOT_USE'])
+        iflag = np.where( (bad1==1) & (bad2 ==1))
+
+        # locate places that flat_dq is set to DO_NOT_USE and reset to UNRELIABLE FLAT
```

Comment:
After discussions with James Muzerolle on how to combine  the s-flat, d-flat and f-flat  (see JP-1663 Aug 31 - Sept 1) 
the 3 flats are independent of each other. Just because 1 flat has an DO_NOT_USE value it does not mean that the combined flat should be flagged as DO_NOT_USE.  It should only be DO_NOT_USE if all 3 flats are do not use. So if one of the flats was flagged as do_not_use but the others were not we changed the final DQ value to be UNRELIABLE_FLAT